model/patch/patch.go

```diff
@@ -136,6 +136,14 @@ type GitMetadata struct {
 	GitVersion string `bson:"git_version,omitempty" json:"git_version,omitempty"`
 }
 
+type Include struct {
```

Comment:
mapping would also let us simplify this [logic](https://github.com/evergreen-ci/evergreen/pull/8005/files#diff-36ad6f3611325fe22762cd371e3130049e163c1ab97d3c55bd011f8827ba0a35R657-R658) since we can index the map directly.

Reply:
I left the logic alone but instead made a `parserInclude` for the parser project because we can't use the model package in the patch package. I would prefer saving this all in a struct because it's easy to pass it around but let me know if you feel strongly about it!

Reply:
nit: i would also call this struct LocalModuleInclude or etc similar to how you named the variable, since that's what it represents